### PR TITLE
fix: loosen filter items tests to accomodate shifting pandas impl

### PR DESCRIPTION
**Pull request by TrevorBergeron**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: tests/system/small/test_dataframe.py [hunk 1]
```diff
@@ -2129,10 +2129,10 @@ def test_df_columns_filter_items(scalars_df_index, scalars_pandas_df_index):
     bf_result = scalars_df_index.filter(items=["string_col", "int64_col"]).to_pandas()
 
     pd_result = scalars_pandas_df_index.filter(items=["string_col", "int64_col"])
-
+    # Ignore column ordering as pandas order differently depending on version
     pd.testing.assert_frame_equal(
-        bf_result,
-        pd_result,
+        bf_result.sort_index(axis=1),
```
Comment (milkshakeiii) [inline tests/system/small/test_dataframe.py:2134]: Testing comments.

**Comment by milkshakeiii** [comment]
LGTM

**Review by milkshakeiii — APPROVED** [review]

**Comment by ashleyxuu** [comment]
Could you rename the commit message following the convention: http://conventionalcommits.org

**Comment by TrevorBergeron** [comment]
> Could you rename the commit message following the convention: http://conventionalcommits.org

done

**Pull request merged**